src/services/navigationBar.ts

```diff
@@ -176,23 +190,35 @@ namespace ts.NavigationBar {
             }
         }
 
+        function hasNamedFunctionDeclarations(nodes: NodeArray<Statement>) {
+            if (forEach(nodes, s => s.kind === SyntaxKind.FunctionDeclaration && !isEmpty((<FunctionDeclaration>s).name.text))) {

```

Comment:
You could instead just return the result of the `forEach`, or just convert this to a for-of loop